src/test/java/org/fogbowcloud/manager/core/cloudconnector/LocalCloudConnectorTest.java

```diff
@@ -90,577 +84,847 @@
 	@Before
     public void setUp() throws FogbowManagerException, UnexpectedException {
 		
```

Comment:
it seems variables declared in lines 65 and 66 are used only by the setUp method. If so, make them local, instead of instance variables